components/viewer/ViewScoreStatus.vue

```diff
@@ -39,7 +39,7 @@ const activeScores = computed<{ score: PointType; value: number }[]>(() => {
   )(scores);
 });
 </script>
-<style land="scss">
+<style scoped land="scss">
```

Comment:
This comes from a conflict that arose from naming the CSS score, I would prefer to keep that name, but I have no issues changing it back to `obj-score` if you'd prefer that naming schema (will also want to change `addon` to `obj-addon` for consistency)